packages/common/browser/algolia/site-search.vue

```diff
@@ -3,6 +3,7 @@
     :index-name="tenantKey"
     :search-client="searchClient"
     :routing="routing"
+    :searchFunction="searchFunction"
```

Comment:
this needs to be :**search-function**="searchFunction"

Reply:
why?

Reply:
Done